dhis-2/dhis-api/src/main/java/org/hisp/dhis/preheat/Preheat.java

```diff
@@ -441,6 +448,30 @@ public boolean isDefault( IdentifiableObject object )
         return defaultObject != null && defaultObject.getUid().equals( object.getUid() );
     }
 
+    public Map<Class<?>, List<Attribute>> getMapAllClassAttributes()
```

Comment:
Better not to expose such maps. I see access does `getMapAllClassAttributes().get( klass )` - For this we should have a `getAttributeByType( Class<? extends IdentifiableObject> klass)`.

Reply:
fixed